src/server-stub/main.rs

```diff
@@ -35,12 +35,28 @@ impl TokenService for Service {
         Ok(Response::new(reply))
     }
 
+    async fn enroll_cluster_agent(
+        &self,
+        request: Request<pb::EnrollClusterAgentRequest>,
+    ) -> Result<Response<pb::EnrollClusterAgentResponse>, Status> {
+
+        println!("enroll_cluster_agent: {:?}", request.into_inner());
+
+        let expiration = SystemTime::now() + Duration::from_secs(180);
+
+        let reply = pb::EnrollClusterAgentResponse{
+            session_token: "SESSION_TOKEN".to_string(),
```

Comment:
Does the session token need to follow the same format as a real one, or is it treated as an opaque blob everywhere?

Reply:
treated as opaque blob. I chose something simple b/c it used to be printed (in this server-stub) and I wanted to see that it looked correct.